src/components/autocomplete/js/autocompleteController.js

```diff
@@ -471,8 +471,8 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
         event.preventDefault();
 
         clearSelectedItem();
-        if ($scope.searchText && hasEscapeOption('clear')) {
-          clearSearchText();
+        if (isTextClearable() && hasEscapeOption('clear')) {
+          clearText();
```

Comment:
We should still stick with `clearSearchText`.

Using `clearText` against `clearSelectedValue` is unclear and also inconsistent